blazor/ClientSideExample/ClientSideExample/ViewModels/GreetingViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace ClientSideExample.ViewModels
+{
+    public class GreetingViewModel : ReactiveObject
+    {
+        public ReactiveCommand<Unit, Unit> Clear { get; }
+
+        [ObservableAsProperty]
```

Comment:
Do we want to use Fody in an official sample?

Reply:
note that GreetingView is the only class to use Fody.  It proves that it works.

Reply:
Removed